osquery/tables/other/augeas.cpp

```diff
@@ -0,0 +1,91 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#include <augeas.h>
```

Comment:
nit, include your non-osquery headers first. This applies to the boost header too, make sure they appear before anything `#include <osquery...`